installation/sources/build-and-install.md

```diff
@@ -73,144 +73,213 @@ it's likely you may need root privileges so you can try to prefixing the command
+Fluent Bit provides certain options to CMake that can be enabled or disabled when configuring, please refer to the following tables under the _General Options_, _Development Options_, Input Plugins _and _Output Plugins_ sections.
 
 ### General Options
 
 | option | description | default |
 | :--- | :--- | :--- |
-| FLB\_ALL | Enable all features available | No |
-| FLB\_JEMALLOC | Use Jemalloc as default memory allocator | No |
-| FLB\_TLS | Build with SSL/TLS support | Yes |
-| FLB\_BINARY | Build executable | Yes |
-| FLB\_EXAMPLES | Build examples | Yes |
-| FLB\_SHARED\_LIB | Build shared library | Yes |
-| FLB\_MTRACE | Enable mtrace support | No |
-| FLB\_INOTIFY | Enable Inotify support | Yes |
-| FLB\_POSIX\_TLS | Force POSIX thread storage | No |
-| FLB\_SQLDB | Enable SQL embedded database support | No |
-| FLB\_HTTP\_SERVER | Enable HTTP Server | No |
-| FLB\_LUAJIT | Enable Lua scripting support | Yes |
-| FLB\_RECORD\_ACCESSOR | Enable record accessor | Yes |
-| FLB\_SIGNV4 | Enable AWS Signv4 support | Yes |
-| FLB\_STATIC\_CONF | Build binary using static configuration files. The value of this option must be a directory containing configuration files. |  |
-| FLB\_STREAM\_PROCESSOR | Enable Stream Processor | Yes |
-| FLB\_CONFIG\_YAML | Enable YAML configuration support | Yes |
-| FLB\_WASM | Build with WASM runtime support | Yes |
-| FLB\_WAMRC | Build with WASM AOT compiler executable | No |
+| FLB_ALL | Enable all features available | No |
+| FLB_ARROW | Build with Apache Arrow support | No |
+| FLB_AVRO_ENCODER | Build with Avro encoding support | No |
+| FLB_AWS | Enable AWS support | Yes |
+| FLB_AWS_ERROR_REPORTER | Build with aws error reporting support | No |
+| FLB_BINARY | Build executable | Yes |
+| FLB_CONFIG_YAML | Enable YAML configuration support | Yes |
+| FLB_CUSTOM_CALYPTIA | Enable Calyptia Support | Yes |
+| FLB_EXAMPLES | Build examples | Yes |
+| FLB_HTTP_SERVER | Enable HTTP Server | No |
+| FLB_INOTIFY | Enable Inotify support | Yes |
+| FLB_JEMALLOC | Use Jemalloc as default memory allocator | No |
+| FLB_LUAJIT | Enable Lua scripting support | Yes |
+| FLB_METRICS | Enable metrics support | Yes |
+| FLB_MTRACE | Enable mtrace support | No |
+| FLB_PARSER | Build with Parser support | Yes |
+| FLB_POSIX_TLS | Force POSIX thread storage | No |
+| FLB_PROXY_GO | Enable Go plugins support | Yes |
+| FLB_RECORD_ACCESSOR | Enable record accessor | Yes |
+| FLB_REGEX | Build with Regex support | Yes |
+| FLB_RELEASE | Build with release mode (-O2 -g -DNDEBUG) | No |
+| FLB_RUN_LDCONFIG | Enable execution of ldconfig after installation | No |
+| FLB_SHARED_LIB | Build shared library | Yes |
+| FLB_SIGNV4 | Enable AWS Signv4 support | Yes |
+| FLB_SQLDB | Enable SQL embedded database support | No |
+| FLB_STATIC_CONF | Build binary using static configuration files. The value of this option must be a directory containing configuration files. |  |
```

Comment:
The other lines have a value for the last cell in the line, but this one is missing a Yes or No.

Reply:
Because there isn't ...
https://github.com/fluent/fluent-bit/blob/4db79b40aa930a29298771e8e4ca4e97d9e9fbba/CMakeLists.txt#L139